src/Common/src/Interop/Unix/libcurl/Interop.libcurl_global.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+using size_t = System.UInt64;
+using curl_socket_t = System.Int32;
+
+internal static partial class Interop
+{
+    internal static partial class libcurl
+    {
+        [DllImport(Interop.Libraries.LibCurl)]
+        public static extern int curl_global_init(
+            long flags);
+
+        [DllImport(Interop.Libraries.LibCurl)]
+        public static extern void curl_global_cleanup();
```

Comment:
How do these differ from the same named functions in Interop.libcurl.cs? https://github.com/dotnet/corefx/pull/2956/files#diff-aea5728308b9f9f8961a847e7a60822dR15